src/main/java/common/tileentities/GTMTE_SOFuelCellMK1.java

```diff
@@ -146,7 +146,7 @@ public boolean checkRecipe(ItemStack stack) {
 					if((liquid = GT_Utility.getFluidForFilledItem(aFuel.getRepresentativeInput(0), true)) != null
 							&& hatchFluid.isFluidEqual(liquid)) {
 
-						liquid.amount = Math.round((EU_PER_TICK * 20) / aFuel.mSpecialValue);
+						liquid.amount = (int) Math.floor((EU_PER_TICK * 20) / aFuel.mSpecialValue);
```

Comment:
As long as the numbers are positive, integer division always rounds down in Java - just do
```java
liquid.amount = (EU_PER_TICK * 20) / aFuel.mSpecialValue;
```

Reply:
Both changes made. I was concerned with the voltage possibly exceeding the dynamo if it matched exactly, during world load, but I don't think that can actually happen.